book/src/advanced-blobs.md

```diff
@@ -0,0 +1,38 @@
+# Blobs
+
+In the Deneb network upgrade, one of the changes is the implementation of EIP-4844, also known as [Proto-danksharding](https://blog.ethereum.org/2024/02/27/dencun-mainnet-announcement). Alongside with this, a new term named `blob` (binary large object) is introduced. Blobs are "side-cars" carrying transactions data in a block. They are mainly used by Ethereum layer 2 operators. As far as stakers are concerned, the main difference with the introduction of blobs is the increased storage requirement. 
+
+### FAQ
+
+1. What is the storage requirement for blobs?
+
+   We expect an additional increase of ~50 GB of storage requirement for blobs (on top of what is required by the consensus and execution clients database). The calculation is as below:
+
+   One blob is 128 KB in size. Each block can carry a maximum of 6 blobs. Blobs will be kept for 4096 epochs and pruned afterwards. This means that the maximum increase in storage requirement will be:
+
+   ```
+   128 KB / blob * 6 blobs / block * 32 blocks / epoch * 4096 epochs = 96 GB
+   ```
+
+   However, in practice, it is expected that not all blocks will be full of blobs. A practical scenario is that each block contains 3 blobs on average, which translates to an increase of storage requirement of 48 GB.
```

Comment:
EIP-1559 style blob gas pricing will guarantee the average to be 3 unless someone can spend infinite ETH in fees.

Reply:
Fix in [b9cd6c7](https://github.com/sigp/lighthouse/pull/5364/commits/b9cd6c7247d47a152e421ed7d79a46863ee6b3ed)

I try to avoid using the word "guarantee" as it sounds a bit too solid. If the revision is not correct let me know